api/src/test/java/org/pmiops/workbench/ras/RasLinkServiceTest.java

```diff
@@ -236,23 +235,6 @@ public void setUp() throws Exception {
     accessModules = TestMockFactory.createAccessModules(accessModuleDao);
   }
 
-  @Test
-  public void testLinkRasIdMeSuccess() throws Exception {
```

Comment:
@Qi77Qi This test is still needed, for Id.Me.  Looks like there's one below here that can be removed, `testLinkRasSuccess_withEraCommons()`